deno/payloads/v10/oauth2.ts

```diff
@@ -111,4 +111,10 @@ export enum OAuth2Scopes {
 	 * See https://discord.com/developers/docs/interactions/application-commands
 	 */
 	ApplicationsCommandsUpdate = 'applications.commands.update',
+	/**
+	 * Allows your app to update permissions for its Application Commands using a bearer token - client credentials grant only
```

Comment:
This is not client credentials only 🤔

Reply:
https://github.com/discord/discord-api-docs/pull/4830/files#diff-d557cf0682f63e6ebb8f6646617ea3c17e359a5fc0f204e99a6980642d785edaR32

Reply:
Well, I just copied and formated from the upstream repository.
![image](https://user-images.githubusercontent.com/79790819/165631696-deeea243-eab4-4e24-9699-439af70cb93b.png)